Foundation/NSURLSession/ftp/FTPURLProtocol.swift

```diff
@@ -0,0 +1,165 @@
+//
+//  FTPURLProtocol.swift
+//  Foundation
+//
+//  Created by sai hema on 6/27/17.
+//  Copyright © 2017 Apple. All rights reserved.
```

Comment:
This should be replaced with the standard header found in other Foundation files.